loadbalancer/algorithm.go

```diff
@@ -30,13 +30,17 @@ const (
 
 	// ConsistentHash indicates choice between the backends based on their hashed address.
 	ConsistentHash
+
+	// PowerOfRandomNChoices selects N random endpoints and picks the one with least outstanding requests from them.
+	PowerOfRandomNChoices
```

Comment:
Please have a look at discussion [regarding naming](https://github.com/zalando/skipper/pull/1507#discussion_r472289180)

Reply:
I agree about name of algorithm is misleading. But according to the paper http://www.eecs.harvard.edu/~michaelm/postscripts/handbook2001.pdf official name is this.

Reply:
I would rather go with the current one, than diverging too much from the original paper